CMakeLists.txt

```diff
@@ -1,15 +1,21 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.16.3)
 
-project(unofficial-discord-client LANGUAGES CXX)
+project(unofficial-discord-client
+	VERSION 1.0
+    DESCRIPTION "Lib to build its daemon"
```

Comment:
VERSION and DESCRIPTION are not consistent

Reply:
Good catch! My bad.